tests/test_reverse_string.py

```diff
@@ -0,0 +1,33 @@
+from unittest import TestCase, main
+
+from task_2 import reverse_string, NonStringInputError
+
+
+class TestReverseString(TestCase):
+    def test_reverse_string(self):
+        # Given
+        cases = [
+            ("abcd efgh", "dcba hgfe"),
+            ("a1bcd efg!h", "d1cba hgf!e"),
+            ("", ""),
+        ]
+        # When
+        for input_text, expected_result in cases:
+            with self.subTest(input_text=input_text):
+                # Then
+                self.assertEqual(reverse_string(input_text), expected_result)
+
+    def test_NonStringInputError(self):
```

Comment:
In python we do use snake_case for naming stuff,  for example this_is_snake_case. You named this function using camelCase convention, thisIsCamelCase
Please rename and follow the convention further

Reply:
Ah and I forgot, return type annotation is missing for both test functions, please add

Reply:
Theretofore you mentioned it I thought it would be appropriate to name the test  function using the exception name that has the capital letter for every word... 
I've renamed test and added annotation